docker/AIDockerImage/requirements.txt

```diff
@@ -0,0 +1,51 @@
+aiohappyeyeballs==2.6.1 ; python_version >= "3.12" and python_version < "4.0"
+aiohttp==3.12.11 ; python_version >= "3.12" and python_version < "4.0"
```

Comment:
Any dependencies with locked dependencies (dependency not set simply to *) needs an explanation to why is it locked